recipes/cutadapt/meta.yaml

```diff
@@ -1,28 +1,32 @@
-{% set version = "1.18" %}
+{% set version = "2.0" %}
 
 package:
   name: cutadapt
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/fd/4d/3ac2947d36e2d56ce1513dc4a53d3a45e520d7ccfb43af9ba85408e44c69/cutadapt-1.18.tar.gz
-  sha256: 17aabf9b19d09a426d96030a83ad003c97b26dba9d45bf5570d33088fcd533f9
+  url: https://files.pythonhosted.org/packages/f2/bd/c4054b10cff0a595f6343edf874b98d6bf568703f0bacb0bfd2967ef662a/cutadapt-2.0.tar.gz
+  sha256: e2f94acdcba745de8c9c44106e99b22975439c9a7ec236aa85f7c3baea839fd1
 
 build:
-  number: 1
+  number: 0
   script: pip install --no-deps .
```

Comment:
Same here, can you adjust it to http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip%20install#use-pip